agent/grpc-external/services/resource/write_test.go

```diff
@@ -46,10 +46,6 @@ func TestWrite_InputValidation(t *testing.T) {
 			artist.Id.Name = ""
 			return artist
 		},
-		"no data": func(artist, _ *pbresource.Resource) *pbresource.Resource {
-			artist.Data = nil
-			return artist
-		},
```

Comment:
should we add a test that empty data is allowed?

Reply:
done